### ISSUE TITLE
Opening fold on home screen opens to middle of app list. 

### ISSUE THREAD

**Issue by NickM-27** [8bitPit/Niagara-Issues#2041]
### Describe the bug

When viewing niagara launcher on the home screen (favorite apps visible) and then unfolding the device, niagara refreshes the middle of the app list (focused on letter P)

### Steps to reproduce

Open launcher on cover screen
Unfold device
Observe that list is scrolled down to letter P (instead of showing favorite apps)

### Device name

Galaxy Fold Z4

### Android version

Android 12

**Comment by NickM-27** [comment]
Looks like this is fixed, can't reproduce

**Comment by Amorphous404** [comment]
Hi @NickM-27, Glad to know you longer face this issue. Thanks for the update.

**Comment by Maxr1998** [comment]
Nope, I think we fixed it. We can close this issue if @NickM-27 also can not reproduce it anymore.
